app/components/stripe-cc-form/component.js

```diff
@@ -0,0 +1,60 @@
+import Ember from 'ember';
+import { createStripeToken } from 'diesel/utils/stripe';
+
+export default Ember.Component.extend({
+  store: Ember.inject.service(),
+  error: null,
+
+  name: null,
+  number: null,
+  cvc: null,
+  expMonth: null,
+  expYear: null,
+  zip: null,
+
+  isSaving: false,
+
+  organization: null,
+  plan: Ember.computed.reads('organization.plan'),
+
+  actions: {
+    saveCC() {
+      let {
+        name,
+        number,
+        cvc,
+        zip,
+        plan,
+        organization,
+        expMonth: exp_month,
+        expYear: exp_year
```

Comment:
some new usage of destructuring assignment. left side is property name from the object (the result of the `getProperties` call), right side is local variable name.